### PR TITLE
Update: Data  and Tools Sections

### PR DESCRIPTION
**Pull request by Claurt07**
Changes: 

Complete revamp of the Tools and Data section based on the wireframes for the upcoming website. The idea is to use the current website to test these wireframes, have time to adapt them and make the transition to the new website easier. 
**New pages within Tools and Data:**

* **Open Mapping Tools** with 6 sections
  * Imagery and Earth Observation 
  * Data Collection: Field 
  * Data Collection: Remote 
  * Data Analysis and OSM Insights 
  * Data Insights: Visualization 
  * OSM Data Access and Sharing 
* **Tech Product Suite** with 5 pages
  * TM 
  * fAIr 
  * HOT Export Tool 
  * OAM 
  * FMTM 

Screenshots of the change: 

image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by Claurt07** [comment]
Hey @kshitijrajsharma this is a big one, lmk and I can walk you through if needed.

**Comment by kshitijrajsharma** [comment]
Sure , Lets go through the meet we have tommorow

File: _layouts/open-data-solutions.html [hunk 1]
```diff
@@ -0,0 +1,1632 @@
...
...gin-inline-end:1rem}.md\:flex{display:flex}}.rtl\:text-right:where([dir="rtl"], [dir="rtl"] *){text-align:right}@media (prefers-color-scheme: white){.dark\:border-gray-700{--tw-border-opacity:1;border-color:rgb(55 65 81 / var(--tw-border-opacity))}.dark\:bg-blue-600{--tw-bg-opacity:1;background-color:rgb(37 99 235 / var(--tw-bg-opacity))}.dark\:bg-gray-800{--tw-bg-opacity:1;background-color:rgb(255 255 255 / var(--tw-bg-opacity))}.dark\:text-blue-500{--tw-text-opacity:1;color:rgb(59 130 246 / var(--tw-text-opacity))}.dark\:text-gray-400{--tw-text-opacity:1;color:rgb(156 163 175 / var(--tw-text-opacity))}.dark\:text-gray-500{--tw-text-opacity:1;color:rgb(107 114 128 / var(--tw-text-opacity))}.dark\:text-white{--tw-text-opacity:1;color:rgb(0 0 0 / var(--tw-text-opacity))}.dark\:hover\:bg-gray-700:hover{--tw-bg-opacity:1;background-color:rgb(55 65 81 / var(--tw-bg-opacity))}.dark\:hover\:bg-gray-800:hover{--tw-bg-opacity:1;background-color:rgb(155 155 155 / var(--tw-bg-opacity))}.dark\:hover\:text-white:hover{--tw-text-opacity:1;color:rgb(255 255 255 / var(--tw-text-opacity))}.dark\:focus\:ring-gray-800:focus{--tw-ring-opacity:1;--tw-ring-color:rgb(31 41 55 / var(--tw-ring-opacity))}}
```
Comment (kshitijrajsharma) [inline _layouts/open-data-solutions.html]: I guess this css is not being used in the html , Can you verify ? if yes then may be we can move it to css file and import it

File: _layouts/open-data-solutions.html [hunk 2]
```diff
@@ -0,0 +1,1632 @@
...
...27.87496 42.00816,27.56536 42.00816,27.15016 L42.00816,20.77816 C42.00816,20.36136 41.84736,20.05256 41.56656,19.85576 C41.28416,19.65816 40.94896,19.52536 40.56576,19.52536 C40.18096,19.52536 39.84736,19.65816 39.56576,19.85576 C39.28256,20.05256 39.05136,20.36136 39.05136,20.77816 L39.05136,27.15016 Z"/><polygon points="33.379 17.815 33.379 22.99 30.152 22.99 30.152 17.815 28.387 17.815 28.387 30.1 30.152 30.1 30.152 24.923 33.379 24.923 33.379 30.1 35.278 30.1 35.278 17.815"/><polygon points="52.431 17.815 45.426 17.815 45.426 19.622 48.015 19.622 47.985 30.1 49.847 30.1 49.847 19.622 52.431 19.622"/><path d="M65.42344,28.7668 L79.67944,28.7668 L71.15544,19.1684 L62.66904,9.57 L54.18104,0.000400000001 L11.53944,0.000400000001 L20.02584,9.57 L5.73784,9.57 L14.22584,19.1684 L14.29144,19.2332 L0.000239999999,19.2332 L8.52184,28.8316 L17.01304,38.43 L25.50024,48.0004 L68.14184,48.0004 L59.65064,38.43 L59.58824,38.3364 L73.87864,38.3364 L65.42344,28.7668 Z M39.83944,39.9492 C30.31864,39.9492 22.60104,32.8084 22.60104,24.0012 C22.60104,15.1908 30.31864,8.05 39.83944,8.05 C49.36104,8.05 57.07944,15.1908 57.07944,24.0012 C57.07944,32.8084 49.36104,39.9492 39.83944,39.9492 Z"/></g></svg>
```
Comment (kshitijrajsharma) [inline _layouts/open-data-solutions.html:120]: svg I suppose ?

File: _layouts/open-data-solutions.html [hunk 3]
```diff
@@ -0,0 +1,1632 @@
...
+                                                    {% for tool in page['Oam'].Tools %}
+                                                      <li><a class="external" href="{{ tool.URL }}" style="color: black;">{{ tool.Name }}</a></li>
+                                                    {% endfor %}
+                                                  </ul>
+                                                </div>
+                                              </div>  
+                                            </div>
+
+
+                                            <!-- <div class="p-6 bg-gray-50 text-medium text-gray-500 dark:text-gray-400 dark:bg-gray-800 rounded-lg w-full" id="open-aerial-map" role="tabpanel" aria-labelledby="dashboard-tab">
```
Comment (kshitijrajsharma) [inline _layouts/open-data-solutions.html]: You can remove this comment

File: _layouts/open-data-solutions.html [hunk 4]
```diff
@@ -0,0 +1,1632 @@
...
+                                                            <li><a class="external" href="{{ tool.URL }}" style="color: black;">{{ tool.Name }}</a></li>
+                                                          {% endfor %}
+                                                        </ul>
+                                                      </div>
+                                                    </div>
+                                                </div>  
+
+                                           
+                                                <div class="p-6 rounded-lg w-full" id="mapbox" role="tabpanel" aria-labelledby="profile-tab">
+                                                  <h3 class="text-lg font-bold mb-2" style="color: #d63f3f; font-family: Barlow Condensed, sans-serif; font-weight: bold; text-transform: uppercase; letter-spacing: .045em;">Mapbox</h3>
```
Comment (kshitijrajsharma) [inline _layouts/open-data-solutions.html]: Should mapbox be listed here ?

File: _layouts/open-data-solutions.html [hunk 5]
```diff
@@ -0,0 +1,1632 @@
...
+                                                      Raw API
```
Comment (kshitijrajsharma) [inline _layouts/open-data-solutions.html]: Raw Data API

File: _layouts/product-pages.html [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+
+          </div>
+        </div>    
+      </div>
+    </div>
+
+
+
+
+    <!-- <div class="program-body" style="padding-top: 1px">
```
Comment (kshitijrajsharma) [inline _layouts/product-pages.html]: Cleanup needed for the comments

File: _layouts/open-data-solutions.html (hunk 1 above)
Comment (Claurt07) [reply _layouts/open-data-solutions.html]: It is being used for the hover effects -- sorry I forgot to put away. Moved out.

File: _layouts/open-data-solutions.html (hunk 2 above)
Comment (Claurt07) [reply _layouts/open-data-solutions.html:120]: yes, idk why for these I chose svg, should I go for <i> instead?

File: _layouts/open-data-solutions.html (hunk 4 above)
Comment (Claurt07) [reply _layouts/open-data-solutions.html]: no, they should not you are right -- deleted

**Comment by Claurt07** [comment]
Thanks for the review @kshitijrajsharma -- addressed all your comments. Thank you again for taking the time :star2:

**Pull request merged**